src/styles/createTheme.ts

```diff
@@ -46,6 +47,7 @@ export const createTheme = ({
     typography: createTypography(typography),
     boxShadows: createBoxShadows(boxShadows),
     pxToRem: (size: number) => `${size / 16}rem`,
+    hexToRgba: hexToRgba,
```

Comment:
Just an FYI, you can also use the shorthand since the key and function are the same name. 

```suggestion
    hexToRgba,
```

Reply:
Ah, thank you! I'll update that if I get back in here; I do usually prefer the shorthand